<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gen7 Time Finder 0.0.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabSelection">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabStationary">
       <attribute name="title">
        <string>Stationary</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="0" colspan="3">
         <widget class="QTableView" name="tableViewStationary">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxStationaryRNGInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>RNG Info</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="labelStationaryStartDate">
             <property name="text">
              <string>Start Date:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="4">
            <widget class="QDateTimeEdit" name="dateTimeEditStationaryStartDate">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="minimumDateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="maximumDate">
              <date>
               <year>2099</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelStationaryEndDate">
             <property name="text">
              <string>End Date:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="4">
            <widget class="QDateTimeEdit" name="dateTimeEditStationaryEndDate">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2010</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="maximumDateTime">
              <datetime>
               <hour>23</hour>
               <minute>59</minute>
               <second>59</second>
               <year>2099</year>
               <month>12</month>
               <day>31</day>
              </datetime>
             </property>
             <property name="minimumDateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelStationaryFrame">
             <property name="text">
              <string>Frame:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="lineEditStationaryStartFrame">
             <property name="text">
              <string notr="true">478</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3" colspan="2">
            <widget class="QLineEdit" name="lineEditStationaryEndFrame">
             <property name="text">
              <string notr="true">10000</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBoxStationaryDelay">
             <property name="text">
              <string>Consider Delay</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="2" colspan="2">
            <widget class="QLineEdit" name="lineEditStationaryDelay">
             <property name="text">
              <string notr="true">0</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QLabel" name="labelStationaryDelay">
             <property name="text">
              <string>+4f</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="5">
            <widget class="QPushButton" name="pushButtonStationarySearch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="5">
            <widget class="QPushButton" name="pushButtonStationaryCancel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBoxStationaryFilter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Filters</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>GroupBox</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QProgressBar" name="progressBarStationary">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabEvent">
       <attribute name="title">
        <string>Event</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0" colspan="2">
         <widget class="QTableView" name="tableViewEvent">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxEventRNGInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>RNG Info</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBoxEventFilter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Filters</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QProgressBar" name="progressBarEvent">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWild">
       <attribute name="title">
        <string>Wild</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0" colspan="2">
         <widget class="QTableView" name="tableViewWild">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBoxWildFilter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Filters</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxWildRNGInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>RNG Info</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QProgressBar" name="progressBarWild">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabID">
       <attribute name="title">
        <string>ID</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0" colspan="2">
         <widget class="QTableView" name="tableViewID">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBoxIDFilter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Filters</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="2">
            <widget class="QLabel" name="labelIDTSV">
             <property name="text">
              <string>TSV</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" rowspan="4">
            <widget class="QTextEdit" name="textEditIDFilter"/>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButtonIDTID">
             <property name="text">
              <string>TID</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="radioButtonIDTIDSID">
             <property name="text">
              <string>TID/SID</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="0" column="3" rowspan="4">
            <widget class="QTextEdit" name="textEditTSVFilter"/>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioButtonIDSID">
             <property name="text">
              <string>SID</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QRadioButton" name="radioButtonIDTID7">
             <property name="text">
              <string>G7TID</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QProgressBar" name="progressBarID">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxIDRNGInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>RNG Info</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="labelIDStartDate">
             <property name="text">
              <string>Start Date:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelIDEndDate">
             <property name="text">
              <string>End Date:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelIDFrame">
             <property name="text">
              <string>Frame:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditIDStartFrame">
             <property name="text">
              <string notr="true">1132</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="lineEditIDEndFrame">
             <property name="text">
              <string notr="true">10000</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QPushButton" name="pushButtonIDSearch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="3">
            <widget class="QPushButton" name="pushButtonIDCancel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QDateTimeEdit" name="dateTimeEditIDStartDate">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="minimumDateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="maximumDate">
              <date>
               <year>2099</year>
               <month>12</month>
               <day>31</day>
              </date>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QDateTimeEdit" name="dateTimeEditIDEndDate">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2010</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="maximumDateTime">
              <datetime>
               <hour>23</hour>
               <minute>59</minute>
               <second>59</second>
               <year>2099</year>
               <month>12</month>
               <day>31</day>
              </datetime>
             </property>
             <property name="minimumDateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
